lib/webpacker/runner.rb

```diff
@@ -0,0 +1,22 @@
+module Webpacker
+  class Runner
+    def self.run(argv)
+      $stdout.sync = true
+
+      new(argv).run
+    end
+
+    def initialize(argv)
+      @argv = argv
+
+      @app_path          = File.expand_path(".", Dir.pwd)
```

Comment:
Using `Dir.pwd` has pros and cons: we can install the executables as binstubs with bundler (which won't supply the application dir context) but we wouldn't be able to execute the binstubs from anywhere but the project root.

Reply:
Yeah... maybe you can leverage the root_path accessor set via webpacker itself which defaults to Rails.root?  Not sure what's best in this context?

Reply:
`Rails.root` would make it easy but I was hoping to avoid relying on loading rails since these are just thin wrapper around node executables.

Reply:
Some alternatives:

* Re-instate the bin templates (backing out of the the bundler binstub change) so we can pass in the app path to the runner 
* Support an ENV var like `APP_PATH` to override the default

Reply:
Yeah, lets tackle that in another PR